source/features/hide-zero-packages.tsx

```diff
@@ -6,7 +6,7 @@ import getTabCount from './remove-projects-tab';
 
 async function init(): Promise<void | false> {
 	const packagesTab = await elementReady([
-		'.BorderGrid-cell a[href$="/packages"]', // `isRepoRoot`
+		'.BorderGrid-cell a[href*="/packages"]', // `isRepoRoot`
```

Comment:
This would also catch any repo that has the word “packages” in it, doesn’t it?

Reply:
You're right, didn't think about that. Then how about
```suggestion
		'.BorderGrid-cell a[href*="/packages?"]', // `isRepoRoot`
```